webcomponents/tool/src/main/frontend/packages/sakai-rubrics/src/SakaiRubricReadonly.js

```diff
@@ -8,14 +8,36 @@ export class SakaiRubricReadonly extends SakaiRubric {
   static properties = {
 
     rubric: { type: Object },
+    isSuperUser: { type: Boolean },
     enablePdfExport: { attribute: "enable-pdf-export", type: Boolean },
   };
 
   constructor() {
 
     super();
-
+    this.isSuperUser = false;
     this.enablePdfExport = false;
+    this.checkSuperUser();
```

Comment:
This is a really bad idea. If we have 100 shared rubrics on a page, that's 100 fetch requests. Instead, you should think in terms of setting is-super-user on the actual tag as an attribute and supplying that once. Look in the RubricsController and look at how enablePdfExport is supplied from the controller. You will need to set that attribute on the list and shared list components so they can then set it on the readonly rubric tags. But, that is a lot better than firing off 100s of fetch requests.